community/captagent/APKBUILD

```diff
@@ -2,13 +2,14 @@
 # Maintainer: Kaarle Ritvanen <kaarle.ritvanen@datakunkku.fi>
 pkgname=captagent
 pkgver=6.3.0
-pkgrel=5
+pkgrel=6
 pkgdesc="HEP Capture Agent for HOMER"
 url="https://github.com/sipcapture/captagent"
 arch="all"
 license="GPL-3.0-or-later"
 makedepends="autoconf automake bison expat-dev flex flex-dev json-c-dev
-             libgcrypt-dev libpcap-dev libtool libuv-dev linux-headers libressl-dev"
+             libgcrypt-dev libpcap-dev libtool libuv-dev linux-headers openssl-dev"
+depends="flex-libs"
```

Comment:
Not needed because linked by build
```
>>> captagent*: Running postcheck for captagent
>>> WARNING: captagent*: Found OpenRC directory (/etc/conf.d or /etc/init.d) but name doesn't end with -openrc
>>> captagent*: Preparing package captagent...
>>> captagent*: Stripping binaries
>>> captagent*: Scanning shared objects
>>> captagent*: Tracing dependencies...
	flex-libs
	so:libc.musl-x86_64.so.1
	so:libexpat.so.1
	so:libfl.so.2
	so:libjson-c.so.4
	so:libpcap.so.1
	so:libuv.so.1
```